nucypher/cli/literature.py

```diff
@@ -102,6 +102,22 @@
 See https://docs.nucypher.com/en/latest/guides/staking_guide.html
 """
 
+#
+# Minting
+#
+
+NO_MINTABLE_PERIODS = "No periods that can be rewarded."
```

Comment:
```suggestion
NO_MINTABLE_PERIODS = "There are no periods that can be rewarded."
```

It's difficult to communicate the right thing here, separate from collect-reward.

Reply:
✔️